dataprep/data_connector/schema.json

```diff
@@ -145,6 +148,9 @@
                     {
                         "type": "boolean"
                     },
+                    {
```

Comment:
The fields definition is the definition of a bunch of fields. Currently it means the fields def can be:
```
{
    "fieldA": "template", // a string template
    "fieldB": false // not required
    "fieldC": {
    // a complete field definition
    }
}
```
Please do not change the fields definition.